lib/scope.js

```diff
@@ -235,6 +235,7 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
                     ref.reference(options);
                 });
                 node.thedef = def;
+                node.reference(options);
```

Comment:
My mistake. I didn't know there was precedence for this. 

Is `options` ever subsequently referenced by the symbol definition?